frontend/app/src/people/widgetViews/summaries/wantedSummaries/codingBounty.tsx

```diff
@@ -125,7 +125,7 @@ function MobileView(props: CodingBountiesProps) {
       if (count >= 29 || data.invoiceStatus) {
         clearTimeout(pollTimeout);
         setPollCount(0);
-        main.setLnInvoice("");
+        main.deleteLnInvoice(main.lnInvoice);
```

Comment:
The only issue I have here is if the user navigates away from the frontend before the polling ends then we still have the same issue, the invoice should be cleared on the backend after how many second we decided.

If a user creates an invoice, then prior to the polling finishing closes the browser, the invoice will still be in the backend cache

Reply:
Read the store documentation, and I realized that the store automatically purges itself, so I have the invoice store expiration to 2 minutes. I have removed the delete functionality since the store will automatically delete it.